src/components/ContactLinks/ContactLinks.scss

```diff
@@ -0,0 +1,38 @@
+@import "styles/colors.scss";
+
+.contact-links {
+  .title {
+    font-size: 2.5em;
+    font-weight: 350;
+  }
+
+  .text {
+    font-size: 1.5em;
+    font-weight: 300;
+    word-break: break-word;
```

Comment:
We should probably use tailwind for word-break

Reply:
Resolved